### PR TITLE
merge feature/rds-phase-2 to develop

### PR DESCRIPTION
**Pull request by jennyhliu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     "@babel/preset-react": "^7.16.5",
     "@babel/register": "^7.16.5",
     "@babel/runtime": "^7.16.5",
-    "@cumulus/api": "10.1.0",
+    "@cumulus/api": "^10.1.1-alpha.7",
```
Comment (kkelly51) [inline package.json:71]: This will be going to v11?

Comment (Jkovarik) [reply package.json:71]: Yeah this isn't correct.    Will fix.

File: app/src/js/utils/table-config/reconciliation-reports.js [hunk 2]
```diff
@@ -252,3 +253,38 @@ export const tableColumnsGnf = [
...
+  },
+  {
+    Header: 'Provider',
+    accessor: 'provider'
+  },
+  {
+    Header: 'Conflict Type',
+    id: 'conflictType'
+  },
+  // {
```
Comment (kkelly51) [inline app/src/js/utils/table-config/reconciliation-reports.js:280]: Are we keeping this Conflict Details section commented for a reason?

Comment (jennyhliu) [reply app/src/js/utils/table-config/reconciliation-reports.js:280]: The display isn't working on the PR, and my new PR for the display ticket will reorganize the fields, and add more information.

**Review by kkelly51 — APPROVED** [review]
Tested 👍

**Pull request merged**